apps/myaccount/src/components/account-recovery/options/sms-recovery/sms-recovery.tsx

```diff
@@ -0,0 +1,436 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { FinalForm, FinalFormField, FormRenderProps, FormValue, TextFieldAdapter } from "@wso2is/form";
+import { Button, GenericIcon, Hint, PrimaryButton, SecondaryButton } from "@wso2is/react-components";
+import { FormValidation } from "@wso2is/validation";
+import { AxiosError } from "axios";
+import isEmpty from "lodash-es/isEmpty";
+import React, { ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch, useSelector } from "react-redux";
+import { AnyAction } from "redux";
+import { ThunkDispatch } from "redux-thunk";
+import { Form, Grid, Icon, List } from "semantic-ui-react";
+import { updateProfileInfo } from "../../../../api";
+import { getAccountRecoveryIcons } from "../../../../configs";
+import { CommonConstants } from "../../../../constants";
+import { commonConfig } from "../../../../extensions";
+import { AlertInterface, AlertLevels, BasicProfileInterface, MultiValue, ProfileSchema } from "../../../../models";
+import { AppState } from "../../../../store";
+import { getProfileInformation, setActiveForm } from "../../../../store/actions";
+import { EditSection } from "../../../shared";
+import "./sms-recovery.scss";
+
+/**
+ * SMS key.
+ */
+const SMS: string = "sms";
```

Comment:
Let's rename the constant to improve the readability. adding more context can make it even more descriptive
Suggestion: `SMS_FORM_KEY`

Reply:
Addressed with https://github.com/wso2/identity-apps/commit/f82c47681cfe094407e95e1696b645a3aa799ef5